### DISCUSSION TITLE
Syncronizing webcam audio and video.

### DISCUSSION THREAD

**Discussion by AlexCrosby** [aiortc/aiortc#605] [Q&A]
I have been playing around with the webcam example, trying to add the webcam audio.
So far I have managed it by adding a second MediaPlayer specifically for the audio, the issue is that the audio has a significant  delay. I have tried playing around with the options such as adding "ac" : "1", "ar" : "16000", but all this has achieved is stretching the audio making everything slow and low pitched.

Is there an easy way / options to reduce the audio delay as much as possible or am I going about this completely the wrong way?

**Comment by Mohit-Ak** [comment]
Can you please post the python code here? I am facing a similar issue.

**Comment by issa-ai** [comment]
Hi,

I am facing the same problem with audio. It is always delayed approximately 2 seconds. Is there a working example of a video and audio stream working in sync or only audio stream with as little delay as possible ?

**Comment by jinqinn** [comment]
i am facing a same isue.
